contrib/masterfiles-stage/common.sh

```diff
@@ -108,6 +108,8 @@ git_deploy_refspec() {
   # and https://github.com/cfengine/core/pull/2465#issuecomment-173656475
   git --git-dir="${local_mirrored_repo}" --work-tree="${temp_stage}" checkout -q -f "${2}^0" ||
     error_exit "Failed to checkout '$2' from '${local_mirrored_repo}'"
+  git --git-dir="${local_mirrored_repo}" --work-tree="${temp_stage}" submodule init"
```

Comment:
I didn't mean to include this, and didn't test it, but for the cfbs case below I had to be in the work tree dir for the commands to work. So maybe just add the `cd` and `cd -` as below and it will work. Will try later.

Reply:
Found this reference:
https://stackoverflow.com/questions/31871888/what-goes-wrong-when-using-git-worktree-with-git-submodules